Client/qtTeamTalk/preferences.ui

```diff
@@ -442,18 +442,45 @@
              </widget>
             </item>
             <item>
-             <widget class="QToolButton" name="chatTemplateToolButton">
-              <property name="text">
-               <string>Edit Chat Message Templates</string>
-              </property>
-             </widget>
+             <layout class="QHBoxLayout" name="horizontalLayout_34">
+              <item>
+               <widget class="QCheckBox" name="chatTemplateChkBox">
+                <property name="text">
+                 <string>Setup text message templates</string>
+                </property>
+               </widget>
+              </item>
+              <item>
+               <widget class="QToolButton" name="chatTemplateToolButton">
+                <property name="accessibleDescription">
+                 <string>Edit Chat Message Templates</string>
+                </property>
+                <property name="text">
+                 <string>...</string>
+                </property>
+               </widget>
+              </item>
+              <item>
+               <spacer name="horizontalSpacer_12">
+                <property name="orientation">
+                 <enum>Qt::Orientation::Horizontal</enum>
+                </property>
+                <property name="sizeHint" stdset="0">
+                 <size>
+                  <width>40</width>
+                  <height>20</height>
+                 </size>
+                </property>
+               </spacer>
+              </item>
+             </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_30">
               <item>
                <widget class="QLabel" name="timestampformatLabel">
                 <property name="text">
-                 <string>Timestamp format</string>
+                 <string>Text message timestamp format</string>
```

Comment:
Why this change? This doesn't affect only text message

Reply:
What else is time stamped?

Reply:
The format defined here affects all date and time, E.G. on files list, user accounts, banned users...

Reply:
Ah, ok. I'll change it back.